Log Viewing/ARKLogTableViewController.m

```diff
@@ -481,10 +481,6 @@ - (void)_scrollTableViewToBottomAnimated:(BOOL)animated;
 
 @implementation ARKTimestampLogMessage
 
-@synthesize text = _text;
```

Comment:
Interesting. Pretty sure I synthesized these because the compiler told me I had to. But if it's working, go for it.